chirp/drivers/radioddity_r2.py

```diff
@@ -502,10 +516,22 @@ def set_memory(self, mem):
         # Get a low-level memory object mapped to the image
         _mem = self._memobj.memory[mem.number - 1]
 
-        LOG.warning('This driver may be broken for deleted memories')
+        _rsvd = _mem.reserved.get_raw()
+
         if mem.empty:
+            _mem.set_raw("\xFF" * 13 + _rsvd)
             return
 
+        if self._frs16:
+            if _mem.rx_freq.get_raw() == "\xFF\xFF\xFF\xFF":
+                FRS_FREQ = int(FRS16_FREQS[mem.number - 1] * 1000000)
```

Comment:
Can we store those in Hz? They're probably okay, but storing frequencies like this in a `float` can be problematic because they don't have infinite resolution. That's why `Memory` stores integers for frequencies (except tune step, I know, but they're small).

Reply:
I will give it a shot. Thanks.

Reply:
Okay, not a huge deal as these are known and don't change I guess. But since you have to multiply them anyway, I think you might as well just store them as integers in Hz. I'll put my OCD away now :)